torch/testing/_internal/common_methods_invocations.py

```diff
@@ -994,11 +994,11 @@ def method_tests():
         ('matrix_power', (S, S, S), [3], "n=3"),
         ('matrix_power', (S, S, S), [1], "n=1"),
         ('matrix_power', (S, S, S), [0], "n=0"),
-        ('matrix_power', lambda: random_fullrank_matrix_distinct_singular_value(S), [-1], "n=-1", (),
+        ('matrix_power', lambda dtype, device: random_fullrank_matrix_distinct_singular_value(S), [-1], "n=-1", (),
```

Comment:
I really look forward to getting rid of method_tests so changes like this are easier and more readable.